source/Halibut.Tests/Support/BackwardsCompatibility/PreviousVersions.cs

```diff
@@ -7,7 +7,7 @@ public static class PreviousVersions
         // The earliest release with a stable WebSocket implementation
         static readonly Version v6_0_658_WebSocket_Stability_Fixes = new("6.0.658");
 
-        // The earliest release with a stable streams where communication won't hang sometimes on net6.0
+        // The earliest release with a stable streams where communication won't hang sometimes on net8.0
```

Comment:
Is this comment change necessary? Opinions?

Reply:
Probably not.